langevals/evaluators/langevals/tests/test_select_best_compare.py

```
@@ -17,14 +17,37 @@
     SelectBestCompareEvaluator,
     SelectBestCompareSettings,
     _slot_label,
 )
 
 
+def _capture_rendered_prompt(evaluator, entry) -> str:
+    """Run one evaluation with the LLM mocked and return the user-message
+    (rendered judge prompt) the evaluator sent to `completion`."""
+    captured: list[str] = []
+
+    def capture(**kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for private function `capture`

(ANN202)